src/clan_arena.c

```diff
@@ -744,6 +606,116 @@ void CA_show_greeting(gedict_t *self)
 	}
 }
 
+void CA_UpdateClients(void)
+{
+	gedict_t *p;
+
+	static double lastupdate = 0;
+
+	if (g_globalvars.time - lastupdate < 0.5)
+		return;
+
+	lastupdate = g_globalvars.time;
+
+	for (p = world; (p = find_client(p));)
+	{
+		CA_SendTeamInfo(p);
+	}
+}
+
+// sends player state information to client for use in teaminfo or other hud elements
+void CA_SendTeamInfo(gedict_t *t)
+{
+	int cl, cnt, h, a, shells, nails, rockets, cells, camode, deadtype, timetospawn;
```

Comment:
could you split these into unique declarations? I know it looks ugly, but that is the code format we use.